### ISSUE TITLE
RDX: Not able to install some extensions

### ISSUE THREAD

**Issue by gunamata** [rancher-sandbox/rancher-desktop#4653]
I run into below error while installing some extensions, For example, `rdctl.exe extension install slimdotai/dd-ext:0.8.2`

```
Internal Server Error
{"message":"500 Internal Server Error"}
```

Going by the log messages in the `extensions.log`, it looks like something is not working properly when dealing with compose files in the extension backend. 

Here are the logs from this session:

[20230511-rdx-install-error.zip](https://github.com/rancher-sandbox/rancher-desktop/files/11454897/20230511-rdx-install-error.zip)

**Comment by mook-as** [comment]
> Failed to install extension slimdotai/dd-ext, cleaning up: SpawnError: C:\rancher-desktop\resources\win32\bin\docker-compose.exe exited with code 1 

(Running `docker-compose.exe up`, but the details here isn't relevant)

It's probably related to:
> `time="2023-05-10T14:44:50-07:00" level=warning msg="The \\"HOME\\" variable is not set. Defaulting to a blank string."\n`

But the error is definitely:
> `container slim-ai-dd-ext is unhealthy\n`

**Comment by mook-as** [comment]
Fixing `$HOME` doesn't seem to help that much; the logs still end up with:
> `{"err":"Credentials invalid","level":"warning","msg":"health check failure","op":"Health.Worker","time":"2023-05-12T20:44:02Z"}`

**Comment by gunamata** [comment]
Ok, It looks like a issue specific to the slim.ai extension?

**Comment by mook-as** [comment]
Yes, it appears to be specific to slim.ai (unless we can dig into it and determine something different).

**Comment by gunamata** [comment]
@nunix , Would you be able to loop in your friends from slim.ai to look into this please?
